helm/methode-image-model-mapper/values.yaml

```diff
@@ -4,6 +4,7 @@
 service:
   name: "" # The name of the service, should be defined in the specific app-configs folder.
   hasHealthcheck: "true"
+eksCluster: true
```

Comment:
I think it should be **false** . Probably is typo.

Reply:
fixed